data_structures/binary_trees/words_alphabetical.c

```diff
@@ -0,0 +1,226 @@
+/**
+ * @file
+ * @brief Printing the words contained in a file "file.txt" in alphabetical
+ * order and also their frequencies in to another file "wordcount.txt"
+ */
+#include <ctype.h>
+#include <stdbool.h>
+#include <stdio.h>
+#include <stdlib.h>
+#include <string.h>
+
+/**
+ * Structure defining a node in the binary tree
+ */
+struct Node
+{
+    char *word;
+    unsigned int frequency;
+    struct Node *left;
+    struct Node *right;
```

Comment:
Please add a one-line description of what the variable is for (**see the example below**).

```c
int64_t data = 0;       ///< The value/key of the node.
```

Reply:
Comments has been added